packages/plugin-ext/src/main/browser/custom-editors/paths.ts

```diff
@@ -0,0 +1,250 @@
+/********************************************************************************
+ * Copyright (c) 2020 SAP SE or an SAP affiliate company and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+// copied from https://github.com/Microsoft/vscode/blob/bf7ac9201e7a7d01741d4e6e64b5dc9f3197d97b/src/vs/base/common/paths.ts
```

Comment:
Does that imported code require a CQ?

Reply:
paths.ts is same file as the one already existing in `callhierarchy` package - `packages\callhierarchy\src\common\paths.ts`

Reply:
@danarad05 i believe there is copied code in this PR. I'll help you with the CQ.

Reply:
If we can find a approved CQ that covers that `vscode` file, and we can re-use the exact same version of the file as was approved in the CQ, then we're good. Else we'll need a new CQ.

Reply:
Created CQ: https://dev.eclipse.org/ipzilla/show_bug.cgi?id=23017

Reply:
> [...]  by clicking button "issues addressed, return CQ to IPTeam".
> 
> Until you do, no one will look at it. Thanks!

BTW, you still need to do this even if you attach the code before being asked by the "bot"